package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "start": "node app.js",
     "test:watch": "mocha --watch ./tests/**/*.test.js",
-    "test": "mocha ./tests/**/*.test.js"
+    "test": "mocha ./tests/**/*.test.js",
+    "lint": "eslint"
```

Comment:
You missed something here.

Reply:
@LalitNM What was missed? Code reviews should be as specific as possible 😃

Reply:
@LalitNM can you please point out what I missed

Reply:
The only thing I see that you could add would be a dir for eslint to look. Also, I find Airbnb's rules a little restrictive and overly-aggressive in some areas. I don't know who to the official owner of this project is, but linting rules are opinionated and they should chime in.

As for your PR, for the job to run, you want to tell it where to look, so line 10 would look like this:
`"lint": "eslint ./public/**/*.js ./serve/**/*.js"`

Reply:
Should I change style guide from airbnb to standard one.

Reply:
> Should I change style guide from airbnb to standard one.

Leave it to Airbnb's rules are fine.

Reply:
> Should I change style guide from airbnb to standard one.

Leave it to Airbnb.